layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteNavigationRestService.java

```diff
@@ -106,7 +116,17 @@ public Response createNode(@Parameter(description = "navigation node id")
         return Response.status(Response.Status.UNAUTHORIZED).build();
       }
       NodeState nodeState;
-      nodeState = new NodeState(nodeLabel, null, -1, -1, isVisible ? Visibility.DISPLAYED : Visibility.HIDDEN, null, null);
+      if (isVisible && scheduleVisibility && startScheduleDate != null && endScheduleDate != null) {
+        if (startScheduleDate > endScheduleDate) {
+          return Response.status(Response.Status.BAD_REQUEST)
+                         .entity("end schedule date must be after start schedule date")
+                         .build();
+        } else {
+          nodeState = new NodeState(nodeLabel, null, startScheduleDate, endScheduleDate, Visibility.TEMPORAL, null, null);
+        }
```

Comment:
```suggestion
        if (startScheduleDate > endScheduleDate) {
          return Response.status(Response.Status.BAD_REQUEST)
                         .entity("end schedule date must be after start schedule date")
                         .build();
        } 
          nodeState = new NodeState(nodeLabel, null, startScheduleDate, endScheduleDate, Visibility.TEMPORAL, null, null);
        
```
check also that startDate should be after current date